content/en/users/aai/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Authentication & Authorization
+linkTitle: Authentication & Authorization
+type: docs
+weight: 30
+description: >
+  Identity and access management in the EGI Cloud
+---
+
+## Working with federated resources
+
+[EGI is a federation](https://www.egi.eu/federation/) of
+compute and storage resource providers.
+
+In order for the providers to be able to
```

Comment:
I would move this to active instead of passive form

Reply:
Improved. But I do not know how to rewrite this in active voice, it's too generic.